src/NServiceBus.Extensions.Hosting.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1,5 +1,9 @@
 namespace NServiceBus
 {
+    public class static HostBuilderExtensions
+    {
+        public static Microsoft.Extensions.Hosting.IHostBuilder UseNServiceBus(this Microsoft.Extensions.Hosting.IHostBuilder hostBuilder, System.Func<Microsoft.Extensions.Hosting.HostBuilderContext, NServiceBus.EndpointConfiguration> endpointConfigurationBuilder) { }
+    }
     public class static ServiceCollectionExtensions
     {
         public static Microsoft.Extensions.DependencyInjection.IServiceCollection AddNServiceBus(this Microsoft.Extensions.DependencyInjection.IServiceCollection services, NServiceBus.EndpointConfiguration configuration) { }
```

Comment:
We said this extension method will be moved to another namespace.
What would be a good namespace?

Reply:
Namespace options:
1. NServiceBus.Web
1. NServiceBus.WebHost
1. NServiceBus.Configuration.AdvancedExtensibility (following our [discoverability design principles](https://github.com/Particular/PlatformArchitecture/blob/master/documentation/design-principles.md))
1. _something else_?

Reply:
`NServiceBus.Extensions.ServiceCollection` perhaps ?

Reply:
We went with `NServiceBus.WebHost`. It's OK, can be reviewed once the work settles and before we move forward with an RTM.